AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Events/EditEventCommandHandler.cs

```diff
@@ -76,7 +77,8 @@ public EditEventCommandHandler(AllReadyContext context)
             return await _context.Events
                 .Include(a => a.RequiredSkills)
                 .Include(a => a.VolunteerTasks)
+                .Include(a => a.Location)
```

Comment:
We need to include Location here, otherwise it creates a new location every time